src/lib/tls/tls_policy.h

```diff
@@ -31,6 +31,11 @@ namespace TLS {
 */
 class BOTAN_PUBLIC_API(2, 0) Policy {
    public:
+      /**
+      * Allow ssl key log file
```

Comment:
This should more clearly state what does/does not happen depending on this result:

> If this function returns true, then Callbacks::tls_ssl_key_log_data will be invoked containing secret information for logging purposes.